playwright.config.ts

```diff
@@ -48,19 +48,19 @@ const config: PlaywrightTestConfig = {
     //   },
     // },
 
-    {
-      name: 'webkit',
-      use: {
-        ...devices['Desktop Safari'],
-      },
-    },
+    // {
```

Comment:
Please confirm that this commented config is intended to be merged into main

Reply:
Kishor has confirmed the commented code is intentional

So I will resolve this conversation